src/field_weather.c

```diff
@@ -1104,3 +1104,11 @@ void ResetPreservedPalettesInWeather(void)
 {
     sPaletteColorMapTypes = sBasePaletteColorMapTypes;
 }
+
+bool8 IsWeatherAlphaBlend(void)
+{
+    return (gWeatherPtr->currWeather == WEATHER_FOG_HORIZONTAL) || 
+           (gWeatherPtr->currWeather == WEATHER_FOG_DIAGONAL)   ||
+           (gWeatherPtr->currWeather == WEATHER_UNDERWATER_BUBBLES) ||
+           (gWeatherPtr->currWeather == WEATHER_UNDERWATER);
```

Comment:
Getting mixed messages from you and Jasper here :)

I'm now realizing that I think he meant wrapping the *entire* statement instead of individual conditions. I'm pushing up a version of this with parentheses and your suggestion for the operator placement. I'm fine with whatever y'all decide

Reply:
Oh, I didn't realize Jasper requested the parenthesis. I don't believe that they were used on the rest of the repo, but as it is right now looks good 👍